src/lib/mod.rs

```diff
@@ -85,3 +105,32 @@ pub fn build_lex<TokId: Copy + Eq + TryFrom<usize>>(s: &str) -> Result<LexerDef<
     parse_lex(s)
 }
 
+/// Statically compile all `.l` files in the `src` directory. A file `src/x.l` can then be used in
+/// a crate with `lrlex_mod!(x)`: it exposes a function `lexerdef()` which returns a
+/// [`LexerDef`](struct.LexerDef.html) that can then be used as normal.
+pub fn process_src_dir<TokId: Copy + fmt::Debug + Eq + TryFrom<usize> + TypeName>() -> Result<(), Box<Error>> {
```

Comment:
Maybe document that this changes the environment?

Reply:
I'm confused: it doesn't change the environment (though it does read from the environment).

Reply:
Maybe I misunderstood, I thought somewhere you write an env var `OUT_DIR` and then read it again?